src/matrix.js

```diff
@@ -30,14 +29,62 @@
  * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
  * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ *
+ *
+ * Portions of this source file are taken from WegGL Fundamentals:
+ *
+ * # Copyright 2012, Gregg Tavares.
```

Comment:
Seems unnecessary to have the hash symbols. 

One thing we did for the jQuery one is make it its own comment block so it's clear where it begins and ends. That would probably be good to do here.

Otherwise looks good!

Reply:
Done